src/p528/ValidateInputs.cpp

```diff
@@ -13,21 +13,31 @@
  |                                      + 1 : POLARIZATION__VERTICAL
  |                p	                - Time percentage
  |
+ |       Output:  warnings          - Warning flags
+ |
  |      Returns:  SUCCESS, or validation error code
  |
  *===========================================================================*/
 int ValidateInputs(double d__km, double h_1__meter, double h_2__meter, 
-    double f__mhz, int T_pol, double p)
+    double f__mhz, int T_pol, double p, int* warnings)
 {
+    int rtn = SUCCESS;
+
     if (d__km < 0)
         return ERROR_VALIDATION__D_KM;
 
-    if (h_1__meter < 1.5 || h_1__meter > 20000)
+    if (h_1__meter < 1.5 || h_1__meter > 80000)
```

Comment:
Is using 80km an arbitrary limit just so that you have some sort of upper bound past the warning range of 20km - 80km?

Reply:
Yeah, this is an arbitrary limit, that takes into account various side conversations where folks are interested in the possibility of expanding the model to, but avoids space-based applications (>100km) as space-based link involved more detailed conversations that need to occur first.